src/databricks/labs/ucx/assessment/workflows.py

```diff
@@ -11,13 +11,14 @@ class Assessment(Workflow):
     def __init__(self):
         super().__init__('assessment')
 
-    @job_task(notebook="hive_metastore/tables.scala")
+    @job_task
     def crawl_tables(self, ctx: RuntimeContext):
         """Iterates over all tables in the Hive Metastore of the current workspace and persists their metadata, such
         as _database name_, _table name_, _table type_, _table location_, etc., in the Delta table named
         `$inventory_database.tables`. Note that the `inventory_database` is set in the configuration file. The metadata
         stored is then used in the subsequent tasks and workflows to, for example,  find all Hive Metastore tables that
         cannot easily be migrated to Unity Catalog."""
+        ctx.py4j_table_crawler.snapshot()
```

Comment:
```suggestion
        ctx.tables_crawler.snapshot()
```
(And update the context to supply the correct crawler.)

Reply:
optionally